### PR TITLE
A new task related to Conftest which makes testing Helm charts easier

### PR DESCRIPTION
**Pull request by garethr**
# Changes

This accompanies the Helm Plugin for Conftest. It uses Helm to render
the template before running the result through Conftest.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: conftest/README.md [hunk 1]
```diff
@@ -62,9 +69,60 @@ container step-conftest has failed  : Error
...
+          value: https://github.com/helm/charts.git
+    params:
+    - name: chart
+      value: stable/mysql
+    - name: policy
+      value: stable/mysql/policy
+```
+
+If you apply the above `TaskRun` you can see the output in the `taskrun` logs. For example:
+
```
Comment (ImJasonH) [inline conftest/README.md]: Example is missing?

File: conftest/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Conftest
 
-This task makes it possible to use [Conftest](https://github.com/instrumenta/conftest) within
+These task make it possible to use [Conftest](https://github.com/instrumenta/conftest) within
```
Comment (ImJasonH) [inline conftest/README.md]: typo: These tasks* make

File: conftest/README.md [hunk 3]
```diff
@@ -62,9 +69,60 @@ container step-conftest has failed  : Error
 * **files**: The files to test against the specified policies
 * **policy**: Where to find the policies (_default:_ `policy`)
 * **output**: Which output format to use (_default:_ `stdout`)
+* **args**: An arrag of additional arguments to pass to Conftest (_defaultt `[]`_)
```
Comment (ImJasonH) [inline conftest/README.md]: typo: arrag -> array

File: conftest/README.md [hunk 4]
```diff
@@ -62,9 +69,60 @@ container step-conftest has failed  : Error
...
+
+
+## Inputs
+
+### Parameters
+
+* **chart**: The chart to test against the specified policies (_default:_ `.`)
+* **policy**: Where to find the policies (_default:_ `policy`)
+* **output**: Which output format to use (_default:_ `stdout`)
 * **args**: An arrag of additional arguments to pass to Conftest (_defaultt `[]`)
```
Comment (ImJasonH) [inline conftest/README.md]: arrag here too

File: conftest/README.md (hunk 3 above)
Comment (garethr) [reply conftest/README.md]: and defaultt -> default

**Comment by garethr** [comment]
Looks like this was merged, I'll put up another PR with typo fixes.

**Pull request merged**